app/views/content/ways-to-train.md

```diff
@@ -194,6 +222,11 @@ Courses typically take one year but you may be able to study part-time.
           Takes 2 years
         </div>
       </div>
+      <div="badge yellow">
```

Comment:
```suggestion
      <div class="badge yellow">
```

Reply:
Think you are missing a class:

![image](https://user-images.githubusercontent.com/87642394/148756482-e2896472-5557-4b19-baeb-74ec226db9c7.png)